content/akeneo-event-platform/concepts.md

```diff
@@ -185,45 +185,53 @@ To help identify duplicated events and deal with un-ordered events if it's somet
 
-Still, we will not retry indefinitely:
-- The platform will stop retrying to deliver events emitted more than 8 hours ago.
-- The platform will start to impact the failure rate, which can lead to a subscription suspension.
+Please note that if the rate drops too significantly, the suspension policy will be triggered ([see bellow](/akeneo-event-platform/concepts.html#suspension-policy.html)).
 
 ### Delivery timeout
 
-Specifically for the HTTPS destination type, the delivery timeout ensures that messages are processed within a specified period. Our system expects your endpoint to process the request in **`3 seconds`**. If the request takes longer than this duration, we stop trying to deliver it, the event will enter the retry process.
+Specifically for the HTTPS destination type, the delivery timeout ensures that messages are processed within a specified  time frame. Your endpoint is expected to handle requests within **`3 seconds`**.  If processing exceeds this duration, the event will enter the retry process ([see bellow](/akeneo-event-platform/concepts.html#retry-policy-for-transient-failures.html)).
 
 Under normal circumstances, your HTTPS endpoint must handle the event as fast as possible.
-**Our recommendation** is to put the message in a `queuing system` or in a `database`, and process the event asynchronously.
+**Our recommendation** is to put the message in a `queuing system` or in a `database` for asynchronous processing.
 
 ### Retry policy for transient failures
 
-Retry policies help manage transient failures and guarantee message delivery even when issues occur during a certain amount of time.
-We handle retry back-off internally, and it is not configurable per tenant.
+Our retry policy helps ensure event delivery during transient failures, allowing for multiple attempts to deliver messages when temporary issues arise. The retry back-off mechanism is managed internally and is not configurable on a per-tenant basis.
 
-If your destination is unable to ingest the event, we will try to deliver it again at the following pace:
- - previous delivery attempt + 5 min
- - previous delivery attempt + 10 min
- - previous delivery attempt + 20 min
+If your destination is unable to ingest an event, we will retry deliver as follow:
+ - First retry: 5 minutes after the previous attempt.
+ - Second retry: 10 minutes after the previous attempt.
+ - Third retry: 20 minutes after the previous attempt.
 
-Such retries are made on best effort, only for transient errors and timeouts. The threshold of the suspension policy continues to be computed during the retry process.
+These retries are on a best-effort basis and apply only to transient errors or timeouts. After **three retry attempts**, the message is dropped.
+This type of failure may triggers the suspension policy ([see bellow](/akeneo-event-platform/concepts.html#suspension-policy.html)).
```

Comment:
```suggestion
This type of failure may trigger the suspension policy ([see bellow](/akeneo-event-platform/concepts.html#suspension-policy.html)).
```

Reply:
y'a pas de s après un may ?

Reply:
Nope, pas de "s" après un verbe modal du genre "may" ou "should", il faut mettre la forme basique du verbe :stuck_out_tongue:

Reply:
j'aurai appris un truc :)